packages/vega-util/src/logger.js

```diff
@@ -1,40 +0,0 @@
-function log(method, level, input) {
-  const args = [level].concat([].slice.call(input));
-  console[method].apply(console, args); // eslint-disable-line no-console
-}
-
-export const None  = 0;
-export const Error = 1;
-export const Warn  = 2;
-export const Info  = 3;
-export const Debug = 4;
-
-export default function(_, method, handler = log) {
-  let level = _ || None;
-  return {
-    level(_) {
-      if (arguments.length) {
```

Comment:
In modern code, `arguments` is discouraged in favor of using a rest parameter

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/arguments#description